person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age:, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission || of_age?
+  end
```

Comment:
- [ ] Great job working on this working on this milestone. As per project requirement, `can_use_services?` is to be a `public method`. Kindly make it public.
![image](https://github.com/MMhaidari/school_library/assets/62849872/b66c0695-f053-4e13-b254-4c4738d8ffad)